src/app/api/common/quorum/getQuorum.ts

```diff
@@ -1,34 +1,30 @@
 import { cache } from "react";
-import provider from "@/app/lib/provider";
 import prisma from "@/app/lib/prisma";
 import { ProposalPayload } from "../proposals/proposal";
 import Tenant from "@/lib/tenant/tenant";
+import { TENANT_NAMESPACES } from "@/lib/constants";
 
-/*
-  Retrieve quorum from the governor contract for the supplied proposal,
-  calculating it if not specified
-
-  @dev: only supports optimisim
-*/
 async function getQuorumForProposal(proposal: ProposalPayload) {
   const { namespace, contracts } = Tenant.current();
 
+  // TODO: Andrei - Refactor this using tenant's governor contract type rather than namespace
```

Comment:
This todo isn't addressed, right? I agree it would be nice to refactor to a governor type but we have so few tenants that it's manageable for now.